README.md

```diff
@@ -97,3 +97,5 @@ This repository is designed to help newbies practice using Git and GitHub by cre
 - [Nguyen Aiden](./naidendev/README.md)
 - [WeiJie Zhang](./weijie-zhang/README.md)
 - [Alice Ibrahimovic](./alice/README.md)
+- [Lysander Stewart](./lysander/README.md)
+
```

Comment:
why new line? there's already an empty line that must be kept

Reply:
Sorry , this is my mistake but I deleted this line